foo/bar.js

```diff
@@ -0,0 +1,22 @@
+// This is a sample file for screenshot purposes
+
+
+export function isChicken(term: string): boolean {
+    switch (term) {
+        case "chicken":
```

Comment:
Are you sure we want "chicken" here?

Reply:
Yes, this is what I want here

Reply:
Yeah but "chicken" might not always be the right way to say this.

Reply:
If you look at the specs that's what it says!

Reply:
Which specs are you reading? Can you send a link?